src/reynir_correct/pattern.py

```diff
@@ -590,6 +610,18 @@ def maybe_place(tree: SimpleTree) -> bool:
             )
         )
 
+        # Check use of "vera að" instead of a simple verb
+        cls.ctx_vera_að = {"verb": "@'vera'",}
+        p.append(
+            (
+                "vera", # Trigger lemma for this pattern
+                "VP > [VP > { %verb } (ADVP|NP-SUBJ)? IP-INF > {TO > nhm}]",
```

Comment:
This could then be just `VP > { @'vera' }` instead of the `%verb` macro.

Reply:
Done